.github/workflows/build.yml

```
@@ -91,6 +95,8 @@ jobs:
         with:
           args: 
              "${{ steps.imagemagick.outputs.version }}"
+      - name: Test
+        run: "! dpkg-deb -I ${{ github.workspace }}/imagemagick_${{ steps.imagemagick.outputs.version }}_amd64.deb |grep -qL 'libcdt|libcgraph|libgvc'"
```

same thing here, also better target the dependencies (and hence avoid `Build-Depends`)
```suggestion
        run: dpkg-deb -f ${{ github.workspace }}/imagemagick_${{ steps.imagemagick.outputs.version }}_amd64.deb Depends | grep -qEv 'libcdt|libcgraph|libgvc'
```